src/usb.c

```diff
@@ -399,9 +401,13 @@ void usb_i2c_restore_irqs(const char *i2c_path, bool force)
 
 void usb_hpm_restore_irqs(bool force)
 {
-    if (adt_is_compatible(adt, 0, "J180dAP"))
+    if (adt_is_compatible(adt, 0, "J180dAP")) {
         usb_i2c_restore_irqs("/arm-io/i2c3", force);
-    usb_i2c_restore_irqs("/arm-io/i2c0", force);
+    } else if (adt_is_compatible(adt, 0, "J773gAP")) {
```

Comment:
This needs to be a plain if. The Mac Pro ("J180dAP") has hpm devices on i2c0 and i2c3